src/components/Chat/ChatBubble.tsx

```diff
@@ -6,20 +6,20 @@ import {
   childrenExist,
   createShorthandFactory,
   customPropTypes,
-  UIComponent,
   IRenderResultConfig,
+  UIComponent,
 } from '../../lib'
 import {
-  ComponentVariablesInput,
   ComponentPartStyle,
+  ComponentVariablesInput,
   IComponentPartStylesInput,
 } from '../../../types/theme'
-import { Extendable, ReactChildren, ItemShorthand } from '../../../types/utils'
+import { Extendable, ItemShorthand, ReactChildren } from '../../../types/utils'
 import Avatar from '../Avatar'
 import Layout from '../Layout'
 import Text from '../Text'
 
-export interface IChatMessageProps {
+export interface IChatBubbleProps {
```

Comment:
Not sure how much of this belongs in Stardust, but seems like we might be missing a couple props for message bubbles.

1. How should we handle a succession of messages? It seems to be common UI practice to show those messages different from message with dividers or other messages between them...
**From FB messenger:**
![image](https://user-images.githubusercontent.com/828692/45977744-4d585380-bfff-11e8-8b00-bc9968918c65.png)

**From Teams:**
![image](https://user-images.githubusercontent.com/828692/45977941-d1124000-bfff-11e8-8b49-7b58ff5e54bc.png)

2. What about _Edited_ or _translated_ messages?
**From Telegram:**
![image](https://user-images.githubusercontent.com/828692/45977555-d4f19280-bffe-11e8-91b1-f2bd6fd8112c.png)

**From Teams:**
![image](https://user-images.githubusercontent.com/828692/45977414-614f8580-bffe-11e8-9dbd-b12de260aaca.png)

3. Message actions and important/mentioned indicators.
**From FB messenger:**
![image](https://user-images.githubusercontent.com/828692/45977846-901a2b80-bfff-11e8-8b69-04cb08b1bfb0.png)

**From Teams:**
![image](https://user-images.githubusercontent.com/828692/45977957-da031180-bfff-11e8-9aa2-a280346eeb06.png)